packages/frontend/src/providers/ReactQueryProvider.tsx

```diff
@@ -0,0 +1,45 @@
+import {
+    DefaultOptions,
+    QueryClient,
+    QueryClientProvider,
+} from '@tanstack/react-query';
+import { FC, PropsWithChildren } from 'react';
+
+// used in test mocks
+// ts-unused-exports:disable-next-line
+export const createQueryClient = (options?: DefaultOptions) => {
```

Comment:
this config is our default configuration. I use this + overrides for the mocked `ReactQueryProvider`